### PR TITLE
Proposed simpler PR template

### PR DESCRIPTION
**Pull request by dschaper**
Meta...

Only one checkbox needed now. Easier to have a bot complain if the box isn't checked.

### CODE REVIEW

**Comment by dschaper** [comment]
DCObot should handle those cases.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
 
-- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
-- [ ] I have made only one major change in my proposed changes.
-- [ ] I have commented my proposed changes within the code.
-- [ ] I have tested my proposed changes, and have included unit tests where possible.
-- [ ] I am willing to help maintain this change if there are issues with it later.
-- [ ] I give this submission freely and claim no ownership.
-- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
-- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
+**Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the `development` branch.**
```
Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: This template is used both for the `core` as well as the `web` repo. However, `development` only exists on `core`, on `web` it's named `devel`. Do we need to mention this?

Comment (dschaper) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: We could remove the codeblock from `development` and say 

**PR's against the repository developmental branch.**

Comment (yubiuser) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Sounds good to me.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
-*A detailed list of any necessary changes*
+- **What documentation changes (if any) are needed to support this PR?:**
+
+*Replace this with a detailed list of any necessary changes*
+
+
+---
+**By submitting this pull request, I confirm the following:** 
 
+1. I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
```
Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:30]:
```suggestion
1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,30 +1,37 @@
...
-- [ ] I have commented my proposed changes within the code.
-- [ ] I have tested my proposed changes, and have included unit tests where possible.
-- [ ] I am willing to help maintain this change if there are issues with it later.
-- [ ] I give this submission freely and claim no ownership.
-- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
-- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
+**Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the repositories developmental branch.**
 
-Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/contributing/).
+We enforce the [DCO](https://docs.pi-hole.net/guides/github/contributing/) for all contributions.
```
Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
We enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions. Please [sign all your commits](https://docs.pi-hole.net/guides/github/how-to-signoff/).
```

**Review by yubiuser — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (dschaper) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: I think that will be confusing to have the DCO and the commit signoffs so close to each other.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,30 +1,37 @@
...
 
-- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
-- [ ] I have made only one major change in my proposed changes.
-- [ ] I have commented my proposed changes within the code.
-- [ ] I have tested my proposed changes, and have included unit tests where possible.
-- [ ] I am willing to help maintain this change if there are issues with it later.
-- [ ] I give this submission freely and claim no ownership.
-- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
-- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
+**Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the repositories developmental branch.**
```
Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
**Please make sure you
 1. base your code and PRs against the repositories developmental branch.
 2. [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits as we enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions**
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion

```

Comment (yubiuser) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion

```

**Review by yubiuser — APPROVED** [review]

**Comment by dschaper** [comment]
I think we have addressed all the outstanding issues with this.

**Pull request merged**